### DISCUSSION TITLE
Remove and add a specific element in a graph by choice.

### DISCUSSION THREAD

**Discussion by jibinjoy8897** [clientIO/joint#2575] [Q&A]
### Introduction

I have a paper same as in demo https://www.jointjs.com/demos/database.I have a separate dialog listing tables where i could hide and show the tables and its links. How can i remove and add an element on choice to the paper.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
See the  [viewport](https://resources.jointjs.com/docs/jointjs/v4.0/joint.html#dia.Paper.prototype.options.viewport) paper option.

An example where the `viewport` method is used: https://www.jointjs.com/demos/container, https://www.jointjs.com/demos/collapse-expand
